.github/pull_request_template.md

```diff
@@ -1,19 +1,40 @@
+## **Description**
+
 <!--
-Thanks for your contribution! Take a moment to answer these questions so that reviewers have the information they need to properly understand your changes:
+Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:
+1. What is the reason for the change?
+2. What is the improvement/solution?
+-->
 
-* What is the current state of things and why does it need to change?
-* What is the solution your changes offer and how does it work?
+## **Related issues**
 
-Are there any issues or other links reviewers should consult to understand this pull request better? For instance:
+Fixes:
 
-* Fixes #12345
-* See: #67890
--->
+## **Manual testing steps**
 
-## Examples
+1. Go to this page...
+2.
+3.
 
-<!--
-Are there any examples of this change being used in another repository?
+## **Screenshots/Recordings**
 
-When considering changes to the MetaMask module template, it's strongly preferred that the change be experimented with in another repository first. This gives reviewers a better sense of how the change works, making it less likely the change will need to be reverted or adjusted later.
--->
+<!-- If applicable, add screenshots and/or recordings to visualize the before and after of your change. -->
+
+### **Before**
+
+<!-- [screenshots/recordings] -->
+
+### **After**
+
+<!-- [screenshots/recordings] -->
+
+## **Pre-merge author checklist**
+
+- [ ] I've completed the PR template to the best of my ability
+- [ ] I’ve included tests if applicable
+- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
+
```

Comment:
perhaps include your screenshots requirement in here as well?

Reply:
Good suggestion. Updated here https://github.com/MetaMask/eslint-plugin-design-tokens/pull/19/commits/c720bc515332fbd4568b0f9f294e9344652cb04a